lib/handlers/subscribe.js

```diff
@@ -23,9 +23,12 @@ function SubAck (packet, granted) {
   this.granted = granted
 }
 
-function Subscription (qos, func) {
+function Subscription (qos, func, rh, rap, nl) {
   this.qos = qos
   this.func = func
+  this.rh = rh
+  this.rap = rap
+  this.nl = nl
```

Comment:
Could you add a comment to explain this properties?

Reply:
Will do